api/tests/integration/domain/usecases/create-pro-organization-with-tags_test.js

```diff
@@ -94,6 +101,52 @@ describe('Integration | UseCases | create-pro-organization', function () {
     }
   });
 
+  it('should create organization invitation with role when email is specified', async function () {
```

Comment:
Merci pour le test en plus ! 

Juste une petite correction sur le nom 
```suggestion
  it('should create organization invitation with role when role is specified', async function () {
```